### PR TITLE
Update xkls_holidays.py

### PR DESCRIPTION
**Pull request by mrkgoh**
Corrected muharram holiday for 2021.
Updated thaipusam holiday from 2020 to 2024. source: https://publicholidays.com.my/thaipusam/
Updated deepavali holiday from 2020 to 2024. source: https://publicholidays.com.my/deepavali/
Updated wesak holiday from 2020 to 2024. source: https://publicholidays.com.my/wesak-day/
misc_adhoc holiday to add "2021-06-07" Yang di pertuan agong's birthday

### CODE REVIEW

File: exchange_calendars/xkls_holidays.py [hunk 1]
```diff
@@ -377,10 +378,16 @@
         "2017-10-18",
         "2018-11-06",
         "2019-10-28",
+        "2020-11-14"
```
Comment (gerrymanoim) [inline exchange_calendars/xkls_holidays.py:381]: Missing a `,` here

**Review by gerrymanoim — CHANGES_REQUESTED** [review]

**Comment by gerrymanoim** [comment]
Also can you please fix the style violations in https://github.com/gerrymanoim/exchange_calendars/pull/23/checks?check_run_id=2680627121. 

Thanks for the PR! Happy to approve once that is done.

File: exchange_calendars/xkls_holidays.py (hunk 1 above)
Comment (mrkgoh) [reply exchange_calendars/xkls_holidays.py]: Fixed. Thanks.

**Comment by gerrymanoim** [comment]
Sorry - can you also please resolve your conflict in `exchange_calendars/xkls_holidays.py`? I think it is because I merged your other PR into master.

**Comment by mrkgoh** [comment]
> Sorry - can you also please resolve your conflict in `exchange_calendars/xkls_holidays.py`? I think it is because I merged your other PR into master.

I tried to corrected it. See if it make sense now on your end.

**Comment by mrkgoh** [comment]
received this check error: @github-actions
Benchmark / Performance regression check (pull_request) Failing after 22s — Performance regression check
Details
@github-actions
CI / build-and-test (ubuntu-latest, 3.7) (pull_request) Failing after 28s — build-and-test (ubuntu-latest, 3.7)
Details

How do I resolve it?

**Pull request closed without merging**